packages/components/inputs/radio-input/src/radio-group.js

```diff
@@ -49,15 +49,15 @@ const Group = (props) => {
   });
   if (props.direction === 'inline') {
     return (
-      <div id={props.id}>
+      <div aria-labelledby={props.id}>
```

Comment:
To make the relationship work with label + non-form elements, we have to use `<label id>` and `<div aria-labelledby>`